.eslintrc

```diff
@@ -31,8 +32,7 @@
       "version": "^16.11.0"
     }
   },
-  // includes the typescript specific rules found here: https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules
-  "plugins": ["@typescript-eslint", "react-hooks", "eslint-plugin-react-hooks"],
+  "plugins": ["prettier", "@typescript-eslint", "react-hooks", "eslint-plugin-react-hooks"],
```

Comment:
Hm, I'm surprised that adding this makes a difference. According to https://github.com/prettier/eslint-plugin-prettier#recommended-configuration, the line `"extends": ["plugin:prettier/recommended"]` has the same effect as:

```
{
  "extends": ["prettier"],
  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": "error",
    "arrow-body-style": "off",
    "prefer-arrow-callback": "off"
  }
}
```.

So the only effect this change has is the addition of `plugin:@typescript-eslint/eslint-recommended` to the `extends` array.